### ISSUE TITLE
License + Status Request

### ISSUE THREAD

**Issue by MGatner** [dioscouri/omnipay-cybersource#28]
@eileenmcnaughton @rdiaztushman @dioscouri

I am interested in working with this package. Please provide a license file ASAP, and a status update on any plans to maintain this repo or your willingness to pass it on.

**Comment by eileenmcnaughton** [comment]
@MGatner I'm happy to add you as a maintainer - I think I have access but I'll give the others a couple of days to weigh in.

Development is mostly happening ad hoc here - ie on client requestion

**Comment by MGatner** [comment]
Thanks for the response! Hold off a few as I have some feelers out. Omnipay seems only moderately maintained these days and built for backwards-compatible PHP support, not something I need - I may be switching handlers.

Either way though you should definitely add a LICENSE file - as is people should not be using this repo at all.

**Comment by eileenmcnaughton** [comment]
I think the other library I investigated was https://github.com/Payum/Payum

**Comment by MGatner** [comment]
Yes that is the other one, but Payum is even more "maintainer unstable" https://github.com/Payum/Payum/issues/854

**Comment by eileenmcnaughton** [comment]
Ah - well I'd be interested to hear  your conclusions

**Comment by MGatner** [comment]
Well I think the real issue is that "curated gateways" means a lot of work for maintainers. It's sexy to offer a list of dozens payment gateways consolidated into one API, but that means someone has to update them every time and they will always be limited to the least restrictive breaking change of the factory (Omnipay).

Most of the large payment processing companies have very comprehensive and frequently-updated SDKs and my hunch is that using gateway wrappers to try to make it easier on the developer is actually robbing them of a lot of great support. You lose the flexibility of the monolithic interface, but a developer could establish her own and make it nearly as easy to bridge.
